src/objects/zcl_abapgit_object_form.clas.abap

```diff
@@ -449,6 +449,26 @@ CLASS zcl_abapgit_object_form IMPLEMENTATION.
         tabs         = es_form_data-tabs
         windows      = es_form_data-windows.
 
+    DATA(lv_firstloop) = abap_true.
+    DATA(lt_windows)   = es_form_data-windows.
+    DATA(lt_lines)     = et_lines.
+    SORT lt_windows BY tdwindow.
+
+    REFRESH et_lines.
+    LOOP AT lt_windows INTO DATA(ls_windows).
+      lv_firstloop = abap_true.
+      READ TABLE lt_lines INTO DATA(ls_lines) WITH KEY tdformat = '/W'
+                                                       tdline = ls_windows-tdwindow.
+      LOOP AT lt_lines INTO ls_lines FROM sy-tabix.
```

Comment:
add `ASSERT sy-subrc = 0.`

Reply:
I don't add 'ASSERT sy-subrc = 0.'. 
The reason is that during my tests I found some windows in the form without elements inside (windows is not used anymore but it sill exists. The devlopper ddin't remove it). In this case the result of the ASSERT is a dump.
Using the loop from SY-TABIX there is no risk. If no entry exists in the internal table then the loop is not executed.

if needed I can add the following condition before doing the loop :
       IF line_exists( lt_lines[ tdformat = lv_elt_windows
                                tdline = ls_form_windows-tdwindow ] ).
Christophe